dev/Expander/Expander.idl

```diff
@@ -27,7 +27,7 @@ unsealed runtimeclass Expander : Windows.UI.Xaml.Controls.ContentControl
     [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
     ExpandDirection ExpandDirection{ get; set; };
 
-    event Windows.Foundation.TypedEventHandler<Expander, ExpanderExpandingEventArgs> Expanding;
+    event Windows.Foundation.TypedEventHandler<Expander, ExpanderExpandedEventArgs> Expanded;
```

Comment:
@ejserna @MikeHill @jevansa This is unfortunate. I just noticed that WPF calls this the Expanded event. This is an API change.

Reply:
See the comment https://github.com/microsoft/microsoft-ui-xaml/issues/4390#issuecomment-788335584 for scenarios where Expanding/Collapsing are also needed. If added, these should support cancelation or respect a new property 'IsPinned' allowing apps to override expanding or collapsing.

As this is new functionality I did not include it in the fix. What existed in code should be 'Expanded' and was incorrectly named. How it got past API review I dont know.

Reply:
For now this just makes it consistent with the WPF/Toolkit names for these events and their expected function based on established naming conventions.

This must have been initially missed in the initial review. By behavior, this is the `Expanded` event but was misnamed. @robloo opened a separate feature request to implement the other side of the contract here: https://github.com/microsoft/microsoft-ui-xaml/issues/4407, so don't think we should spend more time on this. Just name it right for now and worry about if more events are added later.

Reply:
@michael-hawker I would think this was brought up here because this is an API-breaking change. I'm not sure how extensively Expander is being used even in preview. Since there are cases where Expanding would be needed as well, if we implemented both we can avoid any API breaking changes.

Reply:
Right, but it's also still in `preview`, so we should be able to break things before release. Otherwise, what's the point of having a preview release if it has to be API stable? That's why it's important to do before 2.6 ships.

Reply:
Followed up on the reasoning for the current naming. This is the pattern we are using in other places in WinUI like IsOpen/Opening. The intention is that we are in the process of expanding and you can delay load the content in the event and have it show up before the expansion animation runs (event happens synchronously in the same layout pass).

Reply:
Makes sense @ranjeshj, though I think in this case the event was still being called after, so that intent wasn't being followed through. I mean in many cases the pair of events exist. Considering this control exists already in WPF and the Toolkit, at least having the pair to make it easier for folks to migrate is useful too.

Reply:
It makes some sense, yes, but the asymmetry in the event pattern isn't good in my opinion. You are designing things for a very specific use case when in fact there are many. I don't have a good gut feeling about the new IsOpen/Opening event convention you outlined above.

In this case I will add back the Expanding event. However, I would also like to add Collapsing. Each will have their own event args so they can be customized in the future as needed.

Reply:
Thanks @robloo. Makes sense. I would avoid adding the extra events just for the sake of symmetry. If there are compelling scenarios (sounds like we do?) for each of those events, we can add them going forward.

Reply:
@robloo Can you revert the change to the idl in this PR. Per current review we want to stay with the "Expanding" name. We can add the Expanded/Collapsing events, but that will need to go through review/different PR. I'd like to unblock the rest of the changes in the PR and get this merged in.

Reply:
Still hope you re-evaluate the event conventions but yes I will revert those changes as discussed above. It will take another day or so, I waited about a week for feedback and now have other things on my plate.

Reply:
I must also fix the event ordering as Collapsed was originally being fired before the state change (same as Expanding). This was another point of confusion.

Reply:
Reverted 'Expanding' -> 'Expanded' event naming change and fixed ordering of events. Marking resolved. Other events will be addressed in a future PR.